.github/workflows/ci-conan-msvc.yml

```diff
@@ -43,11 +43,11 @@ jobs:
           conan create -s build_type=${{ matrix.build_type }} `
             -s compiler.toolset=${{ matrix.compiler_toolset }} `
             -s compiler.version=${{ matrix.compiler_version }} `
-            -o ogre3d:rendersystem_vulkan=${{ matrix.configs }} `
-            -o ogre3d:plugin_glslang=${{ matrix.configs }} `
-            -o ogre3d:component_bullet=${{ matrix.configs }} `
-            -o ogre3d:enable_double=${{ matrix.enable_double }} `
-            -b missing -b outdated . ${{ matrix.version }}@sintef/stable
+            -o ogre3d:with_sdl=${{ matrix.configs == 'fhsim' && 'False' || 'True' }} `
+            -o ogre3d:rendersystem_vulkan=${{ matrix.configs == 'fhsim' && 'False' || 'True' }} `
+            -o ogre3d:plugin_glslang=${{ matrix.configs == 'fhsim' && 'False' || 'True' }} `
+            -o ogre3d:enable_double=${{ ( matrix.configs == 'fhsim' && matrix.version != '13.4.0' ) && 'True' || 'False' }} `
+            -b missing -b outdated . ${{ matrix.version }}@sintef/${env:SINTEF_CHANNEL}
```

Comment:
Usikker på hvorfor du har "env:" her og ikke i gcc-versjon.

Reply:
This is windows-specific syntax